docs/src/components/landing/GetStartedButtons.js

```diff
@@ -36,20 +36,7 @@ export default function GetStartedButtons(props) {
         >
           {primaryLabel}
         </Button>
-        {secondaryUrl ? (
-          <Button
-            href={secondaryUrl}
-            component={Link}
-            noLinkStyle
-            target={'_blank'}
-            variant="outlined"
-            size="large"
-            color="secondary"
-            endIcon={<KeyboardArrowRightRounded />}
-          >
-            {secondaryLabel}
-          </Button>
-        ) : null}
+        <GithubStars />
```

Comment:
As there's another page with a Github star button, couldn't both pages use the same component?

Reply:
They are both using the same component - the `GithubStars` component inside `landing-studio`

Reply:
Ah nevermind I guess I misread the diff in this case.